modules/indexing/src/test/java/org/apache/ignite/internal/cdc/SqlCdcTest.java

```diff
@@ -376,7 +376,7 @@ else if (evt.configuration().getName().equals(USER)) {
     }
 
     /** */
-    static List<List<?>> executeSql(IgniteEx node, String sqlText, Object... args) {
+    public static List<List<?>> executeSql(IgniteEx node, String sqlText, Object... args) {
```

Comment:
Let's not use methods from different components to reduce conflicts in the future. Especially when it's a single liner.

Reply:
I don't think we should copy `executeSql` method all over the test.